spec/support/shared/spree_builders.rb

```diff
@@ -1,5 +1,4 @@
 RSpec.shared_context 'spree_builders' do
-  # TODO: This starts already to be out of hand. Think about fixing this
```

Comment:
Should we move this to REFACTOR.md or just get rid of it? I know you value these.